src/atoms/Text/text.module.scss

```diff
@@ -4,16 +4,17 @@
   }
 }
 
-.semibold {
-  @include typography-semibold;
+.regular {
+  @include typography-regular;
 }
 
 .light {
   @include typography-light;
 }
 
-.regular {
-  @include typography-regular;
+.strong,
```

Comment:
I meant in the base styles. A `strong` tag should never have a different weight than a `.strong`